lib/renderer/react.js

```diff
@@ -13,12 +13,15 @@ const reactExport = module.exports;
  * @param {string} templateLocation located for thetemplate
  * @param {string} templateContentDir where the template content are located
  * @param {string} transpiledTemplateLocation folder for the transpiled code
+ * @param {Boolean} compile Whether to compile the template files or used the cached transpiled version provided by the template in the '__transpiled' folder
  */
-reactExport.configureReact = async (templateLocation, templateContentDir, transpiledTemplateLocation) => {
+reactExport.configureReact = async (templateLocation, templateContentDir, transpiledTemplateLocation, compile) => {
   const outputDir = path.resolve(templateLocation, `./${transpiledTemplateLocation}`);
-  await AsyncReactSDK.transpileFiles(templateContentDir, outputDir, {
-    recursive: true
-  });
+  if (compile) {
+    await AsyncReactSDK.transpileFiles(templateContentDir, outputDir, {
+      recursive: true
+    });
+  }
```

Comment:
Similar to my previous suggested change, since @derberg suggested to not modify the `configureReact` method, you should revoke your changes here as well. 

```suggestion
 
```

Reply:
will do it once i get back home